client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/EmbeddedCompatContinuousQueryTest.java

```diff
@@ -128,7 +128,6 @@ public void testDisallowGroupingAndAggregation() {
       continuousQuery.addContinuousQueryListener(query, listener);
    }
 
-   @Test(enabled = false, description = "Disabled due to https://issues.jboss.org/browse/ISPN-6730")
    public void testContinuousQuery() {
```

Comment:
It is solved by this PR, I think it could solve on master as well https://github.com/infinispan/infinispan/pull/5181/files#diff-8e69cf0ba1704558827ea5f673a74f63